tests/unit/application/models/bestitAmazonPay4OxidTest.php

```diff
@@ -100,6 +100,9 @@ public function testGetIsSelectedCurrencyAvailable()
      */
     public function testIsActive()
     {
+        if ($this->getTestConfig()->getShopEdition() !== 'EE') {
```

Comment:
Why should this test only related to OXID EE? The Amazon payment button is only active and shown if the right payment is set for the current shop (including active Amazon Pay payment method assigned to a shipping), this is also relevant for CE and PE.

Reply:
Because this test is written expecting to have only the EE Version. If this test should work also for CE and PE, then please provide not Enterprise Edition specific test and I will remove this pull request.

Reply:
I set the view usage to false for this test. Should now work also with CE and PE.